source/about_me.html.haml

```diff
@@ -1,20 +1,20 @@
   %br
-  %p As a major soft skill, I have learned how to communicate more effectively what my ideas were and how to implement them, i.e. use less words to express a concept and do so that every member of the team was on my same page.
+  %p As a major soft skill, I have learned how to communicate more effectively what my ideas are and how to implement them, i.e. use fewer words to express a concept and do so, so that every member of the team was on the same page.
   %br
-  %p On the personal side, I know I grew very much into a more mature and responsible individual thanks to the intensive nature of the course. I have pushed myself over my own limits many times and every and each time I got back on my feet and started from where I left, never giving up. I have learned how to ask for help, how to offer my help to others and to never judge or feel judged indipendently on the situation at hand.
+  %p On the personal side, I grew into a much more mature and responsible individual thanks to the intensive nature of the course. I have pushed myself over my own limits many times, and each time I got back on my feet and started from where I left off, never giving up. I have learned how to ask for help, how to offer my help to others and to never judge or feel judged independently of the situation at hand.
   %br
-  %p When it comes to the technical skills I have acquired, I like to think of the course not as the place where I learned how to write code, but as the place where I learned how to approach problems and to work agile, read and make good use of the available documentation and other people’s experiences to my own advantage, make use of important methods as SCRUM and great tools for Continuous Integration, Version Control, Test and Behavior Driven Development, Project Management and much more. I have learned how to do my best to always meet a deadline, how to plan in advance so that my efforts are homogeneously spread during the given amount of time and how to evaluate a problem and put it in a manageable time and effort frame.
+  %p When it comes to the technical skills I have acquired, I like to think of the course not as the place where I learned how to write code, but as the place where I learned how to approach problems and to work in an Agile fashion; read and make good use of the available documentation and other people’s experiences to my own advantage; make use of important approaches such as SCRUM and great tools for Continuous Integration, Version Control, Test and Behavior Driven Development, Project Management and much more. I have learned how to do my best to always meet a deadline; how to plan in advance so that my efforts are homogeneously spread over the time available and how to evaluate a problem and place it in a manageable time and effort frame.
```

Comment:
Also there is potentially a difference between Behaviour Driven Design and Behaviour Driven Development which we could discuss, but perhaps that's splitting hairs ...